uiowa/uiowa
# Removing 175 footer logo

Fixes #5581 

# How to test
- Sync down a site/sites of your choice
- Check update hook runs with no problems
- Check the logo in the footer is now just UIOWA, instead of UIOWA 175 Years, and that clicking the logo will bring you to uiowa.edu
- See that rest of footer elements remain unchanged
